frontend/src/metabase/lib/utils.unit.spec.ts

```diff
@@ -0,0 +1,49 @@
+import MetabaseUtils from "metabase/lib/utils";
```

Comment:
`frontend/test/metabase/lib/utils.unit.spec.js` already contains tests for `compareVersions`.
The new location is more appropriate though.

1. We should keep them in the same place.
2. There is an opportunity to consolidate these new test cases into the old one.

Reply:
@kamilmielnik do you think I should move the old test suite here or the other way around?

Reply:
The old test suite here because:
- we try to keep tests close to code they're testing
- the new one is in TS